fms_to_hf.py

```diff
@@ -20,9 +20,13 @@ def convert_to_hf(model: LLaMA) -> LlamaForCausalLM:
             num_key_value_heads=None if hf_config.kvheads == 0 else hf_config.kvheads,
             num_hidden_layers=hf_config.nlayers,
             intermediate_size=hf_config.multiple_of
-            * round(
-                int(hf_config.hidden_size * hf_config.hidden_grow_factor)
-                / hf_config.multiple_of
+            * (
+                (
+                    int(hf_config.hidden_grow_factor * hf_config.hidden_size)
+                    + hf_config.multiple_of
+                    - 1
+                )
+                // hf_config.multiple_of
```

Comment:
just to confirm... since this is meant to be equivalent to:
```
            self.hidden_dim = multiple_of * (
                (self.hidden_dim + multiple_of - 1) // multiple_of
            )
```
in feedforward.py in fms on main... I think the implication is that `self.hidden_dim` in fms is the same as `int(hf_config.hidden_grow_factor * hf_config.hidden_size)` here? the int here would also be rounding down so hopefully that's correct, though this part was also in the original code here.

Reply:
@nairbv that's correct.  it also confused me at first but yea - you are right about the diff in the naming of hidden_dim.

Reply:
the `hidden_dim` in https://github.com/foundation-model-stack/foundation-model-stack/blob/6440fee8007a18878276e65c73a0372cc24faedf/fms/modules/feedforward.py#L180-L184
might be better called `intermediate_dim`, as typically `hidden_dim` = `embed_dim`.